File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/Java/C/s684030429_comments_removed.c
#include <stdio.h>

int ni() {
    int val;
    scanf("%d", &val);
    return val;
}

void solve() {
    int a = ni();
    int b = ni();
    printf("%d\n", (a - 2 * b > 0) ? a - 2 * b : 0);
}

int main() {
    solve();
    return 0;
} // End of Code

File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/direct/Java/Python/s684030429_comments_removed.py
import sys

class Main:
    def __init__(self):
        self.sc = Scanner()

    def solve(self):
        a = self.ni()
        b = self.ni()
        print(max(0, a - 2 * b))

class Scanner:
    def __init__(self):
        self.sc = sys.stdin

    def next_int(self):
        return int(self.sc.readline())

    def next_long(self):
        return int(self.sc.readline())

    def next_float(self):
        return float(self.sc.readline())

    def next_double(self):
        return float(self.sc.readline())

    def next_line(self):
        return self.sc.readline().strip()

    def next_string_tokenizer(self, s):
        return s.split()

def main():
    main_obj = Main()
    main_obj.solve()

if __name__ == "__main__":
    main()

# End of Code
